mdata/cache/ccache_test.go

```diff
@@ -85,32 +181,32 @@ func TestSearchFromBeginningComplete(t *testing.T) {
 		t.Fatalf("expected to get 4 itergens, got %d", len(res.Start))
 	}
 
-	if res.Start[0].Ts() != 2000 || res.Start[len(res.Start)-1].Ts() != 5000 {
+	if res.Start[0].Ts() != 1005 || res.Start[len(res.Start)-1].Ts() != 1020 {
 		t.Fatalf("result set is wrong")
 	}
 }
 
 func TestSearchFromBeginningIncompleteEnd(t *testing.T) {
 	metric := "metric1"
 	cc := getConnectedChunks(metric)
-	res := cc.Search(metric, 2500, 6000)
+	res := cc.Search(metric, 1006, 1025)
 	if !res.Complete {
-		t.Fatalf("complete is expected to be false")
+		t.Fatalf("complete is expected to be true")
```

Comment:
Good catch. I was only looking at this test and didn't pay attention to the one directly above!